anti-abuse/wordban/wordban.js

```diff
@@ -51,7 +51,7 @@ module.exports = function(core) {
 						var customPhrases = room.params.antiAbuse.customPhrases;
 						if (customPhrases instanceof Array && customPhrases.length !== 0) {
 							filters.custom = new RegExp("\\b" + customPhrases.map(function(e) {
-								return "\\b" + e + "\\b";
```

Comment:
* some rooms from scrollback contains  custom banned words like this, I just figured out during migration.